index.js

```diff
@@ -15,4 +15,23 @@ async function start() {
   console.log('You entered: ' + secretNumber);
   // Now try and complete the program.
   process.exit();
-}
+
+  async function start() {
```

Comment:
You've already got a start function. You should rename one of the start functions or you will get unexpected behaviors.